### PR TITLE
MEPTS-526 | apply fixes on N1 numerator

### PR DESCRIPTION
**Pull request by Edgencio**

### CODE REVIEW

**Comment by JoaoCumbula** [comment]
@Edgencio it Looks good so far, just need to apply the standards in CohortDefinition name.

**Comment by Edgencio** [comment]
method names updated in order to match the standards!

File: api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java [hunk 1]
```diff
@@ -604,30 +604,38 @@ public CohortDefinition getPatientsWhoAreActiveWithNextPickupAs3Months() {
...
     cd.addSearch("scheduled", EptsReportUtils.map(patientsScheduled, mappings));
     cd.addSearch("quarterly", EptsReportUtils.map(quarterly, dispensationMappings));
     cd.addSearch("startOrContinue", EptsReportUtils.map(startOrContinue, dispensationMappings));
     cd.addSearch("completed", EptsReportUtils.map(completed, dispensationMappings));
-
-    cd.setCompositionString("TxCurr AND (scheduled OR quarterly OR startOrContinue) NOT completed");
+    cd.addSearch("pregnant", EptsReportUtils.map(pregnant,pregnantMappings));
+    cd.addSearch("breastfeeding", EptsReportUtils.map(breastFeeding,breastFeedingMappings));
+    cd.addSearch("tbPatients", EptsReportUtils.map(patientsWithTB,tbMappinggs));
+    cd.setCompositionString("TxCurr AND (scheduled OR quarterly OR startOrContinue) NOT (completed OR pregnant OR breastfeeding OR tbPatients");
```
Comment (psmatsinhe) [inline api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:632]:
```suggestion
    cd.addSearch("PregnantAndBreastfeedingAndOnTBTreatment", EptsReportUtils.map(getPregnantAndBreastfeedingAndOnTBTreatment(),
    "endDate=${endDate},location=${location}"));
    cd.setCompositionString("TxCurr AND (scheduled OR quarterly OR startOrContinue) NOT (completed OR PregnantAndBreastfeedingAndOnTBTreatment");
```

Comment (psmatsinhe) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java:632]: @Edgencio please have a look

Comment (Edgencio) [reply api/src/main/java/org/openmrs/module/eptsreports/reporting/library/cohorts/EriDSDCohortQueries.java]: Thanks for the suggestions @psmatsinhe !

**Pull request merged**